packages/expo-updates/ios/EXUpdates/AppLoader/AppLoader.swift

```diff
@@ -392,9 +393,11 @@ open class AppLoader: NSObject {
         return
       }
 
+      let updateManifest = self.updateResponseContainingManifest!.manifestUpdateResponsePart!.updateManifest
+
       if self.erroredAssets.isEmpty {
         do {
-          try self.database.markUpdateFinished(self.updateResponseContainingManifest!.manifestUpdateResponsePart!.updateManifest)
+          try self.database.markUpdateFinished(updateManifest)
```

Comment:
```suggestion
      if self.erroredAssets.isEmpty {
        do {
          let updateManifest = self.updateResponseContainingManifest!.manifestUpdateResponsePart!.updateManifest
          try self.database.markUpdateFinished(updateManifest)
```

if the intention is to make the line shorter, let's keep the new line in the same block. we probably don't need the when `self.erroredAssets.isEmpty` is false